bindings/go/src/fdb/fdb.go

```diff
@@ -195,48 +194,74 @@ func MustGetAPIVersion() int {
 var apiVersion int
 var networkStarted bool
 var networkMutex sync.RWMutex
+var networkRunning sync.WaitGroup
 
 var openDatabases sync.Map
 
-func startNetwork() error {
+// ensureNetworkIsStarted ensures that the API version is set and then starts the internal network event loop, if not already done.
```

Comment:
```suggestion
// ensureNetworkIsStarted starts the internal network event loop, if not already done.
```

This method doesn't check if the API version is set?